lms/services/__init__.py

```diff
@@ -27,20 +27,24 @@
 
 def includeme(config):
     config.register_service_factory(
-        "lms.services.h_api_requests.HAPIRequests", name="h_api_requests"
+        "lms.services.application_instance_getter.ApplicationInstanceGetter",
+        name="ai_getter",
+    )
+    config.register_service_factory(
+        "lms.services.canvas_api.CanvasAPIClient", name="canvas_api_client"
     )
     config.register_service_factory(
         "lms.services.h_api_client.HAPIClient", name="h_api_client"
     )
     config.register_service_factory(
-        "lms.services.application_instance_getter.ApplicationInstanceGetter",
-        name="ai_getter",
+        "lms.services.h_api_requests.HAPIRequests", name="h_api_requests"
     )
     config.register_service_factory(
         "lms.services.launch_verifier.LaunchVerifier", name="launch_verifier"
     )
     config.register_service_factory(
-        "lms.services.canvas_api.CanvasAPIClient", name="canvas_api_client"
+        "lms.services.lis_result_sourcedid.LISResultSourcedIdService",
+        name="lis_result_sourced_id",
```

Comment:
```suggestion
        name="lis_result_sourcedid",
```

I wonder if we should keep this name the same as it is in the LTI spec?

Reply:
To be consistent `LISResultSourcedIdService` should also be imported into the `__init__.py` here and added to its `__all__`. This allows tests to do `from lms.services import FooService`, not caring about the internal structure of `lms.services`, instead of doing `from lms.services.foo_service import FooService`. It'll allow restructuring `lms.services` internally without having to update every test that imports a service to test or mock that service.

This seems to be applied inconsistently by existing service tests though.

Reply:
> I wonder if we should keep this name the same as it is in the LTI spec?

It was a typo, so I'm glad you caught it (again, related to no tests for `includeme` !)

Reply:
> To be consistent LISResultSourcedIdService should also be imported into the __init__.py here and added to its __all__. This allows tests to do from lms.services import FooService, not caring about the internal structure of lms.services, instead of doing from lms.services.foo_service import FooService. It'll allow restructuring lms.services internally without having to update every test that imports a service to test or mock that service.

I'd be happy to do this, but wonder if we should fix all the services in one go (if I understand what you're suggesting correctly), i.e. as a separate PR. Currently, I'm only seeing Exceptions imported in `service's` `__init__.py` — I _am_ looking in the correct place, right?